src/Pim/Bundle/VersioningBundle/Resources/config/guessers.yml

```diff
@@ -4,7 +4,6 @@ parameters:
     pim_versioning.update_guesser.chained.class:           Pim\Bundle\VersioningBundle\UpdateGuesser\ChainedUpdateGuesser
     pim_versioning.update_guesser.contains_products.class: Pim\Bundle\VersioningBundle\UpdateGuesser\ContainsProductsUpdateGuesser
     pim_versioning.update_guesser.variant_group.class:     Pim\Bundle\VersioningBundle\UpdateGuesser\VariantGroupUpdateGuesser
-    pim_versioning.update_guesser.product_value.class:     Pim\Bundle\VersioningBundle\UpdateGuesser\ProductValueUpdateGuesser
```

Comment:
IMO, we should do both. The class could have been removed, but the service changed to use a new one, for instance.